File: Modul-6/Soal-2/PRAK602-2410817320001-NazlaSalsabila.py
n = int(input("Masukkan jumlah ruangan: "))
zetsu = list(map(int, input("Masukkan banyaknya zetsu putih pada setiap ruangan: ").split()))

for i in range(n):
    print(zetsu[i] * (i + 1), end=" ")
print()

File: Modul-6/Soal-2/PRAK602-2410817320001-NazlaSalsabila.c
#include <stdio.h>

int main() {
    int n;
    printf("Masukkan jumlah ruangan: ");
    scanf("%d", &n);
    
    int zetsu[n];
    printf("Masukkan banyaknya zetsu putih pada setiap ruangan: ");
    for (int i = 0; i < n; i++) {
        scanf("%d", &zetsu[i]);
    }
    
    for (int i = 0; i < n; i++) {
        printf("%d ", zetsu[i] * (i + 1));
    }
    printf("\n");
    
    return 0;
}